packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -332,6 +332,17 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     markNeedsTextLayout();
   }
 
+  /// Character used to obscure text if [obscureText] is true.
+  String get obscuringCharacter => _obscuringCharacter;
+  String _obscuringCharacter;
+  set obscuringCharacter(String value) {
+    if (_obscuringCharacter == value) {
+      return;
+    }
+    _obscuringCharacter = value;
+    markNeedsSemanticsUpdate();
```

Comment:
should we use markNeedsLayout instead? The character might have different width.

Reply:
and should assert it the value is valid in case the developers use the class directly.